src/asmcnc/apps/start_up_sequence/start_up_sequence_manager.py

```diff
@@ -123,6 +127,16 @@ def show_user_pro_plus_safety(self):
 		if ('False' in pro_plus_safety or not pro_plus_safety) and os.path.exists(self.m.theateam_path): return True
 		else: return False
 
+	def show_incorrect_shutdown_screen(self):
+		correct_shutdown = (os.popen('grep "correct_shutdown" config.txt').read())
+		if ('False' in correct_shutdown): return True
+		else:
+			if not correct_shutdown:
```

Comment:
have one class that does the actual sudo calls for shutdown, restart and cancel. This function sets the flag to True.
find all instances where the console is reboot or shutdown and call the implemented functions instead.

Reply:
All the file handling and flag re-setting should be done in a, let's call it, ShutdownHandler. It should also have the shutdown functions from console_utils. The start_up_sequence_manager looks the correct_shutdown state up in the ShutdownHandler.

Reply:
Now I know again why I wrote this: start_up_sequence_manager.py is reading the file as well. It should just read the flag from console_utils like 
`if console_utils.correct_shutdown() : do_something...`
Please fix that, so the file handling is encapsulated in console_utils